libs/base/src/lib/components/wallet/PendingTransactions.tsx

```diff
@@ -138,26 +140,32 @@ export const PendingTransaction: FC<{
   const signer = useSigner()
   const { cancel, send } = useTransactionsDispatch()
   const { estimationError, gasLimit, gasPrice, insufficientBalance } = useGas()
+  const [{ appName }] = useBaseCtx()
 
   const [stakeSignatures, setStakeSignatures] = useStakeSignatures()
   const [isStakeSigned, setIsStakeSigned] = useState<boolean>(false)
   const [isStakeSignedForm, setIsStakeSignedForm] = useState<boolean>(false)
   const stakeSignedFunctions = ['compoundRewards', 'stake(uint256)']
 
+  const isGovernance = appName === APP_NAME.GOVERNANCE
```

Comment:
is there a file for common functions ?  , the same logic is used in 2 different files

Reply:
maybe we could put this in the `useBaseCtx`
@chrisjgf @JamesLefrere what do you think?

Reply:
Could do, not too fussed though. Constant is pulled out